src/main/java/ch/njol/skript/structures/StructUsing.java

```diff
@@ -0,0 +1,92 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.structures;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.experiment.Experiment;
+import org.skriptlang.skript.lang.structure.Structure;
+
+@Name("Using Experimental Feature")
+@Description({
+	"Place at the top of a script file to enable an optional experimental feature.",
+	"For example, this might include "
+})
+@Examples({
+	"using 1.21",
+	"using my-cool-addon-feature"
+})
+@Since("2.9")
+public class StructUsing extends Structure {
+
+	public static final Priority PRIORITY = new Priority(15);
+
+	static {
+		Skript.registerSimpleStructure(StructUsing.class, "using <.+>");
```

Comment:
a colon at the end would mean it is a SectionNode

Reply:
true but what if the developer registered it as simple structure? does that give any error or just work?

Reply:
it would fail to parse as simple structures are only tried with simple nodes

Reply:
No I deliberately left the input and the implementation as free as possible so that if somebody has a cool way of using this I'm not blocking that -- you can *technically* parse some kinds of literal expressions and syntax within this (I left the pattern parsing open to extension) so depending on the implementation it may actually be permissible to have something with `()[]?!-+`.

Reply:
Thank you for the info :zap: sounds good to me